imagetest/utils/test_utils.go

```diff
@@ -105,6 +109,34 @@ func GetMetadataHTTPResponse(path string) (*http.Response, error) {
 	return resp, nil
 }
 
+// PutMetadataGuestAttribute sets the guest attribute in the namespace, and returns an error if this operation fails.
+func PutMetadataGuestAttribute(ctx context.Context, namespace, attribute string) error {
+	path := "guest-attributes/" + namespace + "/" + attribute
```

Comment:
Can you use [url.JoinPath()](https://pkg.go.dev/net/url#JoinPath) instead?

Reply:
done